tools/external-adapter/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:12.22.6-alpine
+FROM node:16.13.2
```

Comment:
Was the alpine variant important here for size?

Reply:
using [16-alpine3.15](https://hub.docker.com/layers/node/library/node/16-alpine3.15/images/sha256-0b4d0c4df160b7e9ea5fa8e366e62fc95f0b3a376b4ae4295acc593280330b00?context=explore) image which is `16.13.2` at ~37MB.  versus [16-buster](https://hub.docker.com/layers/node/library/node/16-buster/images/sha256-7c49a64aba86dd483aa874bc0230c07f282e20741a7c66e426970ecafc149a38?context=explore) at ~332MB.